File: Algorithms/Sortings/Python/selection_sort.py
list = [5, 0, -2, 7, 3]

for i in range(0, len(list) - 1):
	min_index = i
	for j in range(i + 1, len(list)):
		if list[min_index] > list[j]:
			min_index = j
	if min_index != i:
		temp = list[min_index]
		list[min_index] = list[i]
		list[i] = temp

print(list)

File: Algorithms/Sortings/Java/selection_sort.java
void selectionSort(int[] arr) {
	for(int i = 0; i < arr.length - 1; i++) {
		int min_index = i;
		for(int j = i + 1; j < arr.length; j++) {
			if(arr[j] < arr[min_index]) {
				min_index = j;
			}
		}
		if(min_index != i) {
			let temp = arr[min_index];
			arr[min_index] = arr[i];
			arr[i] = temp;
		}
	}
}